server/channels/api4/user_test.go

```diff
@@ -6147,6 +6147,7 @@ func TestPromoteGuestToUser(t *testing.T) {
 	}, "promote a guest to user")
 
 	t.Run("websocket update user event", func(t *testing.T) {
+		t.Skip("https://mattermost.atlassian.net/browse/MM-61736")
```

Comment:
Is this related to https://github.com/mattermost/mattermost/pull/29214 or something else?

Reply:
Most likely unrelated. A lot of these failures are unrelated to my original PR, because I sent a revert PR, and even that failed with random failures.